src/audio/component.c

```diff
@@ -251,7 +251,7 @@ int comp_copy(struct comp_dev *dev)
 		ret = dev->drv->ops.copy(dev);
 
 #if CONFIG_PERFORMANCE_COUNTERS
-		perf_cnt_stamp(&dev->pcd, comp_perf_info, dev);
+		perf_cnt_stamp(&dev->pcd, perf_trace_null, dev);
 		perf_cnt_average(&dev->pcd, comp_perf_avg_info, dev);
 #endif
```

Comment:
We are missing some output if we set null for stamp i.e.
```c
#define comp_perf_info(pcd, comp_p)					\
	comp_info(comp_p, "perf comp_copy peak plat %u cpu %u",		\
		  (uint32_t)((pcd)->plat_delta_peak),			\
		  (uint32_t)((pcd)->cpu_delta_peak))

#define comp_perf_avg_info(pcd, comp_p)					\
	comp_info(comp_p, "perf comp_copy samples %u period %u cpu avg %u peak %u",\
		  (uint32_t)((comp_p)->frames),            \
		  (uint32_t)((comp_p)->period),			    \
		  (uint32_t)((pcd)->cpu_delta_sum),			\
		  (uint32_t)((pcd)->cpu_delta_peak))

```
We lose ```plat_delta_peak``` with this change ?

Can you provide the perf log output before and after this change.